selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/AbstractDevice.java

```diff
@@ -596,4 +597,8 @@ public boolean equals(Object o) {
   public int hashCode() {
     return device.hashCode();
   }
+
+  public String getDeviceType() {
+    return deviceType;
```

Comment:
Thank you. I see that you're setting the deviceType to the value of the hardware model.
Is there any special reason to use a different naming?